src/EnergyPlus/WindowManager.cc

```diff
@@ -5393,6 +5348,10 @@ namespace WindowManager {
                 }
             }
         }
+
+        // total absorptance cannot be negative
+        assert(1 - rfp - tfp >= 0.0);
+        assert(1 - rbp - tfp >= 0.0);
```

Comment:
why is this line repeated? copy paste error?

Reply:
You know...just in case the first attempt is a false positive! 😄 

...good catch.

Reply:
Actually, no...it's not the same. The first is the front reflectance, the second is the back reflectance.